src/NzbDrone.Core/DecisionEngine/Specifications/MaximumSizeSpecification.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using NzbDrone.Core.IndexerSearch.Definitions;
+using NzbDrone.Core.Parser.Model;
+using NzbDrone.Core.Configuration;
+using NLog;
+
+namespace NzbDrone.Core.DecisionEngine.Specifications
+{
+    public class MaximumSizeSpecification : IDecisionEngineSpecification
+    {
+        private readonly IConfigService _configService;
+        private readonly Logger _logger;
+
+        public MaximumSizeSpecification(IConfigService configService, Logger logger)
+        {
+            _configService = configService;
+            _logger = logger;
+        }
+
+        public SpecificationPriority Priority => SpecificationPriority.Default;
+        public RejectionType Type => RejectionType.Permanent;
+
+        public Decision IsSatisfiedBy(RemoteEpisode subject, SearchCriteriaBase searchCriteria)
+        {
+            var size = subject.Release.Size;
```

Comment:
In what unit is this size?

Reply:
bytes, always bytes.

Reply:
Thanks, is there a helper to convert to/from MB? or i should do it by dividing/multiplying by hand?

Reply:
`_configService.MaximumSize.Megabytes()` although that's actually Mebibytes.